Add missing colony logo and breadcrumb to landing page

ArmandoGraterol opened the pull request:
I've updated the page headers to allow the breadcrumbs to show without the need for a title, which is the case for the landing page. The changes introduced should be specifically for this, but they are made in general components so make sure to test the breadcrumbs and titles of other pages in order to check that they haven't changed with this.

![FireShot Capture 239 - Welcome to Colony - localhost](https://github.com/JoinColony/colonyCDapp/assets/18473896/e13bcf1d-ea00-416b-bec2-ab51a433bbce)

Resolves #1412

thomassdk reviewed (changes requested):
Code looks good to me but im getting the logo flashing in the top left corner

[Screencast from 2023-11-30 08-35-28.webm](https://github.com/JoinColony/colonyCDapp/assets/32598350/0eca21d5-35e8-45f0-9ba3-1f303a6ca47f)

thomassdk commented:
[Screencast from 2023-11-30 08-36-44.webm](https://github.com/JoinColony/colonyCDapp/assets/32598350/7623df7c-d970-44ea-b266-c2bda8212633)

melyndav reviewed (changes requested):
Heya @ArmandoGraterol! Thanks for picking up this issue. A couple of issues picked up.

1. Padding top and bottom in the nav bar has changed when my wallet is disconnected but isn't an issue when it's connected. The should be the same.

2. There should be no hover or interaction on the title in this state (it should work in other states) for both disconnected and connected states:
![image](https://github.com/JoinColony/colonyCDapp/assets/82127904/48852ecb-e918-484a-b554-1225b221d770)

3. When my wallet is connected, I should see the colony avatr and not this placeholder colony avatar that's been added here by PagePro:

![image](https://github.com/JoinColony/colonyCDapp/assets/82127904/abcda147-684a-4ee1-b7e9-8cf8e488946e)

ArmandoGraterol commented:
@thomassdk

> Code looks good to me but im getting the logo flashing in the top left corner

Unfortunately, changing doesn't seem trivial. This happens because the user is not connected fast enough, that's why it flashes. It happens in other parts as well, for example in one of the items on the landing page.

The first solution that came to my mind was adding a spinner to the landing page, but if we check for `walletConnecting || userLoading` then the spinner will show when we open the modal to connect a wallet, which is terrible in this case, and that is ignoring the fact that we are adding yet another spinner to the CDapp.

I think I'll have to talk about this with Chris to see how can we proceed.

@melyndav
> 1. Padding top and bottom in the nav bar has changed when my wallet is disconnected but isn't an issue when it's connected. The should be the same.
> 2. There should be no hover or interaction on the title in this state (it should work in other states) for both disconnected and connected states:

Fixed!

> 3. When my wallet is connected, I should see the colony avatr and not this placeholder colony avatar that's been added here by PagePro:

There's a separate issue for this: https://github.com/JoinColony/colonyCDapp/issues/1411 which I'm working on right meow.

melyndav reviewed (approved):
Looking slick @ArmandoGraterol! Thanks for your work on this one!

arrenv reviewed (approved):
Looks good my man @ArmandoGraterol !! Even adds it on the users account page.

ArmandoGraterol commented:
@thomassdk I've added a loading spinner to the landing page for when the user or wallet is loading, this should prevent the flashing when reloading with autologin. Before this spinner would even show when opening the dialog to connect a wallet so I've moved the handler to a bit further down the road when you actually click and connect a wallet. Let me know if this is nicer than the previous solution.
